src/types.ts

```diff
@@ -245,12 +245,18 @@ export interface MetaTransactionV1QuoteResponse extends BasePriceResponse {
  * Response type for /meta_transaction/v2/quote endpoint
  */
 export interface MetaTransactionV2QuoteResponse extends BasePriceResponse {
-    metaTransactionHash: string;
-    // TODO(vic): Update to MetaTransactionV2 when it's ready
-    metaTransaction: ExchangeProxyMetaTransaction;
+    metaTransaction: MetaTransactionField;
     fees?: Fees;
 }
 
+type MetaTransactionField = MetaTransactionV1Field /* add meta-transaction v2 when it's ready */;
```

Comment:
Need some suggestions on the name of the type @phil-ociraptor @byeongminP

Reply:
If we were to go for the same nomenclature as `rfq-api`, I'd suggest `MetaTransactionV1Response`